bokehjs/src/lib/models/tools/actions/save_tool.ts

```diff
@@ -24,7 +24,14 @@ export class SaveToolView extends ActionToolView {
   doit(action: "save" | "copy" = "save"): void {
     switch (action) {
       case "save":
-        this.save("bokeh_plot")
+        if (this.model.filename === null) {
```

Comment:
We prefer to use `==`, unless we intend to distinguish between `undefined` and `null`, which isn't the case here.